doc/DeveloperGuide.md

```diff
@@ -94,6 +94,10 @@ Use case ends.
 2. Should be able to hold up to 1000 persons.
 3. Should come with automated unit tests and open source code.
 4. Should favor DOS style commands over Unix-style commands.
+5. The system should be usable to someone familiar with typing commands.
+6. The product is not required to have extravagant GUI effects.
+7. Data should be persistent across sessions.
```

Comment:
This is a functional requirement, not a NFR -- it means a data persistence feature needs to be built.